ompi/mca/osc/rdma/osc_rdma_component.c

```diff
@@ -937,6 +937,23 @@ static int ompi_osc_rdma_query_alternate_btls (ompi_communicator_t *comm, ompi_o
     return btls_found > 0 ? OMPI_SUCCESS : OMPI_ERR_UNREACH;
 }
 
+/* Check for BTL requirements:
+ *  1) RDMA (put/get) and ATOMIC operations.  We only require cswap
+ *     and fetch and add and will emulate other opterations with those
+ *     two as necessary.
+ *  2) Remote Completion
+ *
+ * If the BTL meets all those requirements, increment the btl_counts
```

Comment:
I did not see `btl_counts` increased in this function, maybe this comment is miss placed?

Reply:
Ooops; that was a cut-and-paste issue.  Fixed.